docs/en/sql-reference/data-types/data-types-binary-encoding.md

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
The same is documented in a header DataTypesBinaryeEncoding.h. Providing this as markdown user documentation is probably not useful for anyone?

Reply:
> Providing this as markdown user documentation is probably not useful for anyone?

I want this specification to be presented in the documentation. It can be used in clients for clickhouse in different languages to simplify the parsing of data types in future.